html/index.php

```diff
@@ -263,6 +263,13 @@ function isReferrer($referrer)
         $tech_support_recipient = xd_utilities\getConfiguration('general', 'tech_support_recipient');
         print "CCR.xdmod.tech_support_recipient = CCR.xdmod.support_email = '$tech_support_recipient';\n";
 
+        $tech_support_url = false;
+        try {
+            $tech_support_url = xd_utilities\getConfiguration('general', 'tech_support_url');
+        } catch (exception $ex) {
+        }
+        print "CCR.xdmod.support_url = CCR.xdmod.support_url = '$tech_support_url';\n";
```

Comment:
```suggestion
        print 'CCR.xdmod.support_url = ' . json_encode($tech_support_url) . ";\n";
```

You had a typo in the string: "CCR.xdmod.support_url = CCR.xdmod.support_url".

Also you are going to have to encode the $tech_supprt_url variable to make sure it is valid json. I suggest using json_encode. This will ensure that any special characters in the string are escaped and if the variable is set to `false` then it will get output as javascript `false`. The current code will set a javascript string that has the letters  f a l s e in it. This will evaluate to true!

Reply:
![image](https://user-images.githubusercontent.com/5342179/186521082-79004f27-a72e-4c98-b14a-201a41b12d1a.png)

Reply:
Ok I see your point about using json_encode. Just to clarify, this is to also handle the case where tech_support_url is set as 'false' in the configuration file?

Reply:
The scenario I was thinking of was when `tech_support_url` is absent from the file. Having thought about it I realize that php boolean variable false will evaluate to empty string in your code, which is javascript falsy. So this case won't happen. 

But certainly to need to make sure that you encode the `$tech_support_url` variable properly